eessi/testsuite/tests/apps/openfoam/openfoam.py

```diff
@@ -41,6 +41,17 @@
 from eessi.testsuite.utils import find_modules
 
 
+def filter_scales_64M():
+    """
+    Filtering function for filtering scales for the OpenFOAM 64M mesh test
+    returns all scales with at least half a node.
+    """
+    return [
+        k for (k, v) in SCALES.items()
+        if (v['num_nodes'] >= 4) and (0 < v.get('node_part', 0) <= 2)
```

Comment:
we should change this at some point to have a limit on the number of cores instead of nodes, because our new Tier-1 cluster will have 386 cores per node.

Reply:
In that case, the only choice will be to skip the test based on core count because the core count typically is known after the setup phase. Right now lower scale tests do not get staged.